bing_bert/nvidia/modelingpreln.py

```diff
@@ -541,7 +541,7 @@ def __init__(self, config, args):
                 attn_dropout_checkpoint=args.attention_dropout_checkpoint,
                 normalize_invertible=args.normalize_invertible,
                 gelu_checkpoint=args.gelu_checkpoint,
-                stochastic_mode=True)
+                stochastic_mode=args.stochastic_mode)

```

Comment:
do we need to add this flag in script file for bing-bert, so that it is enabled when kernel is enabled?

Reply:
got it, thanks.